shakenfist_client/commandline/instance.py

```diff
@@ -408,14 +422,27 @@ def instance_create(ctx, name=None, cpus=None, memory=None, network=None, floate
                 return
             video[s[0]] = s[1]
 
+    metadata_def = {}
+    for m in metadata:
+        try:
+            key, val = m.split('=')
+        except ValueError:
+            print('Unable split metadata, correct format is key=value')
```

Comment:
"Unable to"? I wouldn't nit, except its user facing.

Reply:
Good pick up.